src/main/java/me/moodcat/database/entities/Room.java

```diff
@@ -62,12 +60,7 @@
     /**
      * Songs to be played.
      */
-    @ManyToMany(fetch = EAGER)
-    @JoinTable(name = "room_play_queue", joinColumns = {
-            @JoinColumn(name = "room_id", referencedColumnName = "id")
-    }, inverseJoinColumns = {
-            @JoinColumn(name = "song_id", referencedColumnName = "id")
-    })
+    @Transient
```

Comment:
This should not be the way we want to do this. You've just dropped the columns? So then just remove the attribute. A transient column makes no sense. If we want the history and playqueue in the database, there should be tables and they should be eagerly fetched manytomany relations. If we dont want this in the database, then we should remove the attributes from the entity alltogher. This is no fix but rather a clean up the entire model :P